compiler/onecc-docker/onecc-docker

```diff
@@ -16,10 +16,43 @@
 
 import sys
 import subprocess
-import json
 import requests
 import os
 import argparse
+import re
+
+
+class RequestHandler:
+    def __init__(self, token=None, timeout=None):
+        if token:
+            self.headers = {"Authorization": "Bearer {}".format(token)}
+        else:
+            self.headers = {}
+        self.timeout = timeout or 5
+
+    def make_request(self, url):
+        try:
+            response = requests.get(url, headers=self.headers)
+            response.raise_for_status()
+            return response
+        except requests.RequestException as e:
+            raise SystemExit('[onecc-docker] error: {}'.format(e))
```

Comment:
You don't need to `[onecc-docker]` phrase. It will be added in catch function of main.

Reply:
However the original code only shows the message as is. 
The message thrown in the function below shows only what is.
```
except requests.exceptions.RequestException as e:
        raise SystemExit('onecc-docker: error: {}'.format(e))
```

Reply:
Oh, I understood. The original code had a problem. :(
It seems right to raise Runtime error here. I will fix it later. Thank you.